source/material_model/rheology/drucker_prager.cc

```diff
@@ -95,25 +100,24 @@ namespace aspect
         // Initial yield stress (no stabilization terms)
         const double yield_stress = ( (dim==3)
                                       ?
-                                      ( 6.0 * cohesion * cos_phi + 6.0 * pressure * sin_phi) * stress_inv_part
+                                      (( 6.0 * cohesion * cos_phi + 6.0 * pressure * sin_phi) * stress_inv_part) * yield_stress_prefactor
                                       :
-                                      cohesion * cos_phi + pressure * sin_phi);
+                                      (cohesion * cos_phi + pressure * sin_phi)) * yield_stress_prefactor; //added yield_stress_prefactor to calculte the yield stress
```

Comment:
```suggestion
                                      (cohesion * cos_phi + pressure * sin_phi)) * yield_stress_prefactor;
```

When you update the PR, can you remove this and other similar comments?